### PR TITLE
rev

### PR DESCRIPTION
**Pull request by MikeJSeo**

### CODE REVIEW

**Comment by gravesti** [comment]
@MikeJSeo Can you update the code in `data-raw/dummy-anchored.R`? And add an entry in `R/data.R`?

**Comment by MikeJSeo** [comment]
Hey @gravesti 
I put in 1000 bootstrap iterations in weighted_twt. Is that okay?
Any more changes I need to do?

File: R/data.R [hunk 1]
```diff
@@ -208,6 +208,20 @@
 #' @family anchored datasets
 "centered_ipd_twt"
 
+#' Weighted object for two arm trial data
+#' @format A `maicplus_estimate_weights` object created by [estimate_weights()] containing
```
Comment (gravesti) [inline R/data.R:212]: Could you explain which parameters were used in estimate_weights() to generate this object?

File: data-raw/dummy_anchored.R [hunk 2]
```diff
@@ -73,8 +73,11 @@ agd <- process_agd(agd)
 adsl_twt <- dummize_ipd(adsl_twt, dummize_cols = c("SEX"), dummize_ref_level = c("Female"))
 centered_ipd_twt <- center_ipd(ipd = adsl_twt, agd = agd)
 
+centered_colnames <- paste0(c("AGE", "AGE_MEDIAN", "AGE_SQUARED", "SEX_MALE", "ECOG0", "SMOKE"), "_CENTERED")
+weighted_twt <- estimate_weights(data = centered_ipd_twt, centered_colnames = centered_colnames, n_boot_iteration = 1000)
```
Comment (gravesti) [inline data-raw/dummy_anchored.R:77]: Can you see how long it takes to run `maic_anchored()` with 1000 samples? If it's more than a few seconds we should reduce N

File: R/data.R (hunk 1 above)
Comment (MikeJSeo) [reply R/data.R:212]: what do you mean..
estimate_weights(data = centered_ipd_twt, centered_colnames = centered_colnames, n_boot_iteration = 1000)

these are three parameters used..

Comment (gravesti) [reply R/data.R:212]: Can you include that in the roxygen documentation?

File: data-raw/dummy_anchored.R (hunk 2 above)
Comment (MikeJSeo) [reply data-raw/dummy_anchored.R:77]: it takes 8 sec for 1000 samples, 1 sec for 100 samples. Which?

File: R/data.R (hunk 1 above)
Comment (MikeJSeo) [reply R/data.R:212]: #' Weighted object for two arm trial data. **Parameters data, centered_colnames, and n_boot_iteration was used.**
#' @format A `maicplus_estimate_weights` object created by [estimate_weights()] containing
#'   \describe{
#'     \item{data}{patient level data with weights}
#'     \item{centered_colnames}{Columns used in MAIC}
#'     \item{nr_missing}{Number of observations with missing data}
#'     \item{ess}{Expected sample size}
#'     \item{opt}{Information from `optim` from weight calculation}
#'     \item{boot}{Parameters and bootstrap sample weights, `NULL` in this object}
#'   }
#' @keywords dataset
#' @family anchored datasets
"weighted_twt"


?? like this?

File: data-raw/dummy_anchored.R (hunk 2 above)
Comment (gravesti) [reply data-raw/dummy_anchored.R:77]: Better go for 100 then. We will have trouble on CRAN otherwise

Comment (MikeJSeo) [reply data-raw/dummy_anchored.R:77]: okay will run with 100!

File: R/data.R (hunk 1 above)
Comment (MikeJSeo) [reply R/data.R:212]: can you answer this too? :) @gravesti

File: R/data.R [hunk 3]
```diff
@@ -208,6 +208,20 @@
...
+#'   \describe{
+#'     \item{data}{patient level data with weights}
+#'     \item{centered_colnames}{Columns used in MAIC}
+#'     \item{nr_missing}{Number of observations with missing data}
+#'     \item{ess}{Expected sample size}
+#'     \item{opt}{Information from `optim` from weight calculation}
+#'     \item{boot}{Parameters and bootstrap sample weights, `NULL` in this object}
+#'   }
```
Comment (gravesti) [inline R/data.R:220]:
```suggestion
#'     \item{boot}{Parameters and bootstrap sample weights for the 100 samples}
#' }
#' @description
#' The weighted patient data for a two arm trial generated from the centered patient data ([centered_ipd_twt]).
#' It has weights calculated for 100 bootstrap samples.
#'
#' The object is generated using the following code:
#' ```
#' estimate_weights(
#'   data = centered_ipd_twt,
#'   centered_colnames = c(
#'     "AGE_CENTERED",
#'     "AGE_MEDIAN_CENTERED",
#'     "AGE_SQUARED_CENTERED",
#'     "SEX_MALE_CENTERED",
#'     "ECOG0_CENTERED",
#'     "SMOKE_CENTERED"
#'     ),
#'   n_boot_iteration = 100
#'  )
#' ```
#'
```

File: R/data.R (hunk 1 above)
Comment (gravesti) [reply R/data.R:212]: I made a suggestion in another comment now

**Review by gravesti — APPROVED** [review]
Great, thanks @MikeJSeo !

**Pull request merged**